netebpfext/ebpf_ext_attach_provider.h

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+#include <ntddk.h>
+
+#include <ebpf_platform.h>
```

Comment:
```suggestion
#include "ebpf_platform.h"
```
Only use <> for system headers